api/data_explorer/controllers/export_url_controller.py

```diff
@@ -103,23 +107,62 @@ def _get_entities_dict(cohort_name, query):
                 'query': query
             }
         })
+
+    if cohort_name:
```

Comment:
Line 102 should be `if cohort_name`. There should be one `if cohort_name` that includes both cohort and sample_set.

Reply:
Done.